packages/suite/src/components/suite/modals/AddAccount/index.tsx

```diff
@@ -18,7 +18,7 @@ import { AddAccountButton } from './components/AddAccountButton';
 import { useDispatch } from 'react-redux';
 
 const StyledModal = styled(Modal)`
-    min-height: 550px;
+    width: 560px;
```

Comment:
Too narrow

Reply:
Not really, it's more tight that way and easier to visually scan the options, since there are 1-3 coins in a row ;)